src/widget/city_figure.h

```diff
@@ -4,6 +4,12 @@
 #include "figure/figure.h"
 #include "widget/city.h"
 
+enum {
+    FIGURE_HIGHLIGHT_NONE = 0,
+    FIGURE_HIGHLIGHT_RED = 1,
+    FIGURE_HIGHLIGHT_GREEN = 2,
+}highlight_mask;
```

Comment:
```suggestion
};
```

Don't define the enum otherwise you're actually creating a variable.

Also, this doesn't need to be in the header file.